packages/components/src/tag/tests/jest/TagGroup.test.tsx

```diff
@@ -1,33 +1,37 @@
 import { render, screen } from "@hopper-ui/test-utils";
 import { createRef } from "react";
 
-import { Tag, TagGroupContext, TagList, TagGroup } from "../../src/index.ts";
+import { Tag, TagGroup, TagGroupContext, type TagListProps } from "../../src/index.ts";
+
+type ExtendedTagListProps<T extends object> = TagListProps<T> & {
```

Comment:
Why is this needed?

Reply:
Btw this kind of workaround i'd like you to point them to me before I start reviewing.

Reply:
well since the data-testid prop isn't directly on TagList, but is inside tagListProps prop, typescript thinks that it doesn't support data attributes.. so it's basically to shut typescript up without putting a comment to ignore it.